app/models/dimensions/item.rb

```diff
@@ -28,14 +48,15 @@ def quality_metrics_required?(attributes)
     attributes[:locale] == 'en' && attributes[:content_hash] != content_hash
   end
 
-  def self.create_empty(content_id:, base_path:, locale:)
+  def self.create_empty(content_id:, base_path:, locale:, payload_version:)
     create(
       content_id: content_id,
       base_path: base_path,
       locale: locale,
       latest: true,
       outdated: true,
-      outdated_at: Time.zone.now
+      outdated_at: Time.zone.now,
```

Comment:
That would make more sense yeah. Do you think this is useful information to store? Because I was planning to get rid of the `outdated` flag once it's no longer needed by the ETL process.

Reply:
Yes, true, we can get rid of it after we finish this work. 

In any case, it would useful to track when was the Content Item updated (the effective time of the new version), as we shouldn't rely on the fields (`updated_at` and `created_at` of AR). We could rename the attribute in the future.

Reply:
I've looked into this and it seems we don't have a timestamp for the event itself.

The code that generates the message is here:
https://github.com/alphagov/publishing-api/blob/master/app/presenters/edition_presenter.rb#L37

For major/minor/links messages, we will have the `updated_at` of the edition itself, but that is a different event to whatever triggered the message we process. Unpublishing messages won't have a date at all.

Reply:
mmm, this is unexpected. I would expect the events to be self-contained (including the time where they were generated). This is not in any case something to be sorted out in this PR, as this is external to the CPM.